### PR TITLE
A few fixes for python3

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR implements a few fixes required to get hveto to run end-to-end on python3.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -65,15 +65,15 @@
...
-    'dqsegdb',
-    'gwpy >= 0.12.0',
-    'lxml',
     'gwdetchar',  # for omega scans only
+    'gwpy >= 0.12.0',
     'gwtrigfind',
+    'lxml',
+    'lscsoft-glue >= 1.60.0 ; python_version < \'3\'',
+    'lscsoft-glue >= 2.0.0 ; python_version > \'3\'',
+    'matplotlib',
```
Comment (alurban) [inline setup.py]: @duncanmmacleod, I just took your changes out for a spin in a conda environment with python3.7. It works (thank you!) except that I had to install a modern version of `dqsegdb`, one that utilizes `ligo.segments` and not `glue.segmens`. Can you make the following change?

```suggestion
    'matplotlib',
    'dqsegdb >= 1.6.0',
```

Output from my test is [**here**](https://ldas-jobs.ligo-la.caltech.edu/~aurban/hveto/day/20190108) (requires `LIGO.ORG` credentials).

Comment (duncanmmacleod) [reply setup.py]: I held off making that change for now since gwpy-0.14.0 will not require `dqsegdb`, so neither will hveto...

Comment (alurban) [reply setup.py]: Note, my test run finished in 1 minute 20 seconds with `--nproc 16`, compared to something closer to 20 minutes when run with python2.7

Comment (alurban) [reply setup.py]: Aha, gotcha, in that case I revoke my comment.

**Pull request merged**